app/services/check_answers/sections/case_details.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module CheckAnswers
+  module Sections
+    class CaseDetails < Base
+      def initialize(check)
+        super(check:, label: :case_details)
+        @check = check
+      end
+
+      def subsections
+        tables = [level_of_help_table]
+        if @check.under_eighteen?
+          tables += [aggregated_means_table] unless @check.controlled_clr?
+        else
+          tables += if @check.controlled?
+                      [immigration_or_asylum_table]
+                    else
+                      [domestic_abuse_table, immigration_or_asylum_type_upper_tribunal_table]
+                    end
+          if @check.immigration_or_asylum?
+            tables += if @check.controlled?
+                        [immigration_or_asylum_type_table, asylum_support_table]
+                      else
+                        [asylum_support_table]
+                      end
+          end
+        end
+        [Subsection.new(tables:)]
+      end
+
+    private
+
+      # skip_change_link is only set true when screen is set on the field.
+      # Mostly it is set false and screen is nil other than level_of_help and domestic_abuse
+      # where the reverse is true
+      def level_of_help_table
+        Table.new(screen: :level_of_help, skip_change_link: true, index: nil, disputed?: false,
+                  fields: [
+                    FieldPresenter.new(table_label: :level_of_help, screen: :level_of_help, attribute: :level_of_help, type: :select, model: @check),
+                    if @check.controlled?
```

Comment:
@starswan I think we also need to check this is and under 18 case as well. I get this additional table otherwise. 

![Screenshot 2024-08-15 at 10 32 38](https://github.com/user-attachments/assets/a135bc4a-997a-4e82-9e57-28c2bf7c4fd2)